code/datums/storage/subtypes/medkits.dm

```diff
@@ -0,0 +1,142 @@
+///Regular medkit
+/datum/storage/medkit
+	open_sound = 'sound/items/handling/medkit/medkit_open.ogg'
+	open_sound_vary = TRUE
+	rustle_sound = 'sound/items/handling/medkit/medkit_rustle.ogg'
+	max_specific_storage = WEIGHT_CLASS_SMALL
+
+	///List of everything a medkit can hold
+	VAR_FINAL/static/list/obj/item/list_of_everything_medkits_can_hold = list(
+		//surgery tools
+		/obj/item/surgical_drapes,
+		/obj/item/scalpel,
+		/obj/item/circular_saw,
+		/obj/item/bonesetter,
+		/obj/item/surgicaldrill,
+		/obj/item/retractor,
+		/obj/item/cautery,
+		/obj/item/hemostat,
+		/obj/item/blood_filter,
+
+		//special tools
+		/obj/item/healthanalyzer,
+		/obj/item/dnainjector,
+		/obj/item/lazarus_injector,
+		/obj/item/implantcase,
+		/obj/item/implant,
+		/obj/item/implanter,
+
+		//stacks
+		/obj/item/stack/medical,
+		/obj/item/stack/medical/gauze,
+		/obj/item/stack/sticky_tape,
+		/obj/item/stack/medical/suture/medicated,
+		/obj/item/stack/medical/mesh/advanced,
```

Comment:
Is there a particular reason we have `/obj/item/stack/medical/gauze`, `/obj/item/stack/medical/suture/medicated`, and `/obj/item/stack/medical/mesh/advanced` here explicitly when we already have the parent type `/obj/item/stack/medical` directly above?

Reply:
No reason actually since you brought it up  now. I can remove it